SQLScripts/InsertData_ATeam.sql

```diff
@@ -1,14 +1,167 @@
 use Kainos_A;
 
-insert into band (bandName, bandExpectations)  VALUES(("Apprentice", "Understands own strengths and areas of development. Self-aware of own wellbeingand seeks support where appropriate."));
 
-insert into capability (capability) VALUES(("Engineering"), 
+INSERT INTO band(bandName, bandExpectations)
+VALUES ("Apprentice", "");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Trainee", "");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Associate", "");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Senior Associate", "");
+INSERT INTO band(bandName, bandExpectations)
+VALUES ("Consultant", "");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Manager","");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Principle", "");
+INSERT INTO band(bandName,bandExpectations)
+VALUES ("Leadership Community", "");
```

Comment:
This would be a lot more readable as a single `INSERT` with multiple entries for `VALUES`